M/MuJoCo/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MuJoCo"
+version = v"2.3.7"
+
+# Collection of sources required to build mujoco
+sources = [
+    GitSource("https://github.com/deepmind/mujoco", "c9246e1f5006379d599e0bcddf159a8616d31441"),
+    DirectorySource("bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mujoco
+if [[ "${target}" == *-mingw* ]]; then
+    atomic_patch -p1 ../patches/mingw.patch
+elif [[ "${target}" == *-apple-darwin* ]]; then
+    atomic_patch -p1 ../patches/macos.patch
+else
+    atomic_patch -p1 ../patches/other.patch
+fi
+CPPFLAGS="-I${prefix}/include"
+CXXFLAGS="-I${prefix}/include"
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DMUJOCO_BUILD_EXAMPLES=OFF -DMUJOCO_BUILD_SIMULATE=OFF -DMUJOCO_BUILD_TESTS=OFF -DMUJOCO_TEST_PYTHON_UTIL=OFF .
```

Comment:
Optional suggestion, feel free to ignore, but: I personally like to split these into multiple lines, as that makes it (a) more readable, and (b) if any arguments are added/modified/removed later on, it is easier to see in the diff what changed and how
```suggestion
cmake \
  -DCMAKE_INSTALL_PREFIX=$prefix \
  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
  -DCMAKE_BUILD_TYPE=Release \
  -DMUJOCO_BUILD_EXAMPLES=OFF \
  -DMUJOCO_BUILD_SIMULATE=OFF \
  -DMUJOCO_BUILD_TESTS=OFF \
  -DMUJOCO_TEST_PYTHON_UTIL=OFF \
  .
```

Reply:
Thanks for the suggestion, this is much better!